### DISCUSSION TITLE
How to choose a specific custom page to use with single-product on woocommerce?

### DISCUSSION THREAD

**Discussion by fernandofas** [wprig/wprig#719] [General]
Hi All,

I'm developing an ecommerce website and my single-product page is using the index.php template.
I want to use a custom page with the single-product content where I want to customize the css.

I can't find a way to connect a the content with a specific page.

Any help will be appreciated.

Kind regards,

Fernando Fas

**Comment by jacklowrie** [comment]
Where in your theme is your template file? also, what's the template's filename, and do you have any comments at the top of the template?

**Comment by robruiz** [comment]
Usually you just put a single-product.php in the theme and you are good to
go for over-riding the template heirarchy.
